src/main.go

```diff
@@ -22,14 +23,20 @@ func main() {
 	}
 
 	// GET ENVIRONMENT VARIABLES
-	environment := os.Getenv("ENVIRONMENT")
+	environment := strings.ToUpper(os.Getenv("ENVIRONMENT"))
 
 	db_host := os.Getenv(fmt.Sprintf("%s_DB_HOST", environment))
 	db_port := os.Getenv(fmt.Sprintf("%s_DB_PORT", environment))
 	db_user := os.Getenv(fmt.Sprintf("%s_DB_USER", environment))
 	db_password := os.Getenv(fmt.Sprintf("%s_DB_PASSWORD", environment))
 	db_name := os.Getenv(fmt.Sprintf("%s_DB_NAME", environment))
 
+	// Check if environment variables are set
+	if db_host == "" || db_port == "" || db_user == "" || db_password == "" || db_name == "" {
+		log.Fatalf("Required environment variables are not set")
+		os.Exit(1)
+	}
```

Comment:
Wollen wir dafür ne kleine Helper-Funktion schreiben? Sollte auch mit dem CodeScene-Problem helfen. Also im Utils-Package iwie `stringutils.go` und darin so eine Funktion:


```go
package utils

func containsEmptyString(strings ..string) bool {
    for _, s := range strings {
        if s == "" {
            return true
        }
    }
    
    return false
}
```

Und dann sowas wie:

```suggestion
        if containsEmptyString(db_host, db_port, db_user, db_password, db_name) {
		log.Fatalf("Required environment variables are not set")
		os.Exit(1)
	}
```

Reply:
Klingt gut. soll ich das Commit

Reply:
Klingt gut. Kannst du das Commit für mich ausführen, ich müsste jetzt los